Command/DumpCommand.php

```diff
@@ -31,6 +33,13 @@ protected function configure()
             ))
             ->setDescription('Dumps all JS translation files to the filesystem')
             ->addOption(
+                'pattern',
+                null,
+                InputOption::VALUE_OPTIONAL,
```

Comment:
that's wrong. The value is required. Passing `--pattern` without value does not make sense.